File: Elixir/hello_world_again/lib/kex_2/kex_2.ex
defmodule Kex_2 do
  @spec run_kex_2 :: :ok
  def run_kex_2 do
    IO.puts("冒険が今始まる！")
    player = Player.new(Constants.player_initial_hp, Constants.player_initial_defense)
    IO.puts(Kex2_Battle.game_loop(player, 0, 1))
  end
end

File: Erlang/hello_world/src/questions/kex_2/kex_2.erl
-module(kex_2).
-export([run_kex2/0]).
-include("../../shared/player.hrl").

-spec player_initial_hp() -> 500.
player_initial_hp() -> 500.

-spec player_initial_defense() -> 30.
player_initial_defense() -> 30.

-spec run_kex2() -> 'ok'.
run_kex2() ->
    io:format("冒険が今始まる！~n"),
    Player = #player{ hp = player_initial_hp(), defense = player_initial_defense() },
    Result = kex_2_battle:game_loop(Player, 0, 1),
    io:format("~ts", [Result]).